include/ideep/tensor.hpp

```diff
@@ -56,6 +56,16 @@ class tensor : public memory {
       set_g(1);
     }
 
+    // Construct a memory descriptor from a binary blob.
+    // The API is available since oneDNN v3.4.1
+    desc(const std::vector<uint8_t> &blob) : memory::desc(get_dnnl_blob(blob)) {
+      // The groups is serialized into the last sizeof(int) item
+      // TODO: only little endian is supported here.
```

Comment:
We can just say we use little endian encoding here?

Reply:
Comments have been updated.